torch/csrc/distributed/rpc/python_remote_call.cpp

```diff
@@ -42,14 +45,19 @@ std::unique_ptr<PythonRemoteCall> PythonRemoteCall::fromMessage(
   auto values = value.toTuple()->elements();
 
   // remove the last element from values and convert it back to an RRef
+  bool isAsyncExecution = values.back().toBool();
```

Comment:
nit: should we have a TORCH_INTERNAL_ASSERT validating the number of elements needed in the tuple?